Siri/L1/Level1.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+
+public class Level1 {
```

Comment:
It might be better to use a more intuitive class name here! I noticed the same issue in other places too.